lib/game.rb

```diff
@@ -13,14 +14,14 @@ class Game
             :venue_link
 
   def initialize(row)
-    @game_id = row[:game_id]
-    @season = row[:season]
+    @game_id = row[:game_id].to_i
```

Comment:
Do we need game_id, season, away/home_team_id to be integers?

Reply:
Per the Setup, "Turing - Futbol - File I/O, and Statistics" there are methods that have return values of integers and floats, I discovered through testing if you do not initialize the respective attribute variables as integers (instead of strings). Then it will not add the total score as 2+3 = 5. It will add "2" +"5" which will equal "23". We don't want to return 23 as the score becuase 2+3 is equal to 5 not 23. Essentially "2"+"3" is string interpolation not adding integers.